File: 0001_0100/0063/0063.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s, m = (int(x) for x in f.read().split())
    y = int((s + (s * s - 4 * m) ** 0.5) / 2)
    x = s - y
    if x > y:
        x, y = y, x
    q.write('{} {}'.format(x, y))
        
    
           
        

File: 0001_0100/0063/0063.go
package main

import (
	"fmt"
	"math"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s, m int
	fmt.Fscan(f, &s, &m)
	y := (s + int(math.Sqrt(float64(s*s-4*m)))) / 2
	x := s - y
	if x > y {
		x, y = y, x
	}
	fmt.Fprint(q, x, " ", y)
}
